specification/dimensions/chargesubcategory.md

```diff
@@ -0,0 +1,63 @@
+
+When Charge Type is "Usage", the Charge Subcategory specifies if the usage is accounted for by pre-purchased commitments or if the usage has not benefited from these discounts.
+
+When Charge Type is "Adjustment", the Charge Subcategory indicates what kind of after-the-fact adjustment the record represents. Adjustment type is commonly used to identify changes like credits and refunds.
+
+ChargeSubcategory MUST abide by the following requirements:
+
+- The ChargeSubcategory MUST be present in the billing data.
+- ChargeSubcategory is of type String and MUST be one of the allowed values.
+- ChargeSubcategory MUST NOT be null or empty when ChargeType is "Usage" and the charge is covered by a commitment.
+  - When a usage charge is covered by a commitment, ChargeSubcategory MUST be "Commitment Used".
+  - When a commitment is not used within the committed period, ChargeSubcategory MUST be "Commitment Not Used" for the unused usage charge.
+- ChargeSubcategory MUST be null when ChargeType is "Usage" and is not covered by a commitment.
+- ChargeSubcategory MUST NOT be null or empty when ChargeType is "Adjustment".
+  - When ChargeSubcategory is "Refund" or "Credit", the charge MUST be negative.
+  - When an adjustment applies to a specific item, the corresponding FOCUS columns that identify that item MUST NOT be null and MUST match the applicable item details the adjustment pertains to.
+- ChargeSubcategory MUST be null when ChargeType is "Purchase" or "Tax".
+
+## Column ID
+
+ChargeSubcategory
+
+## Display Name
+
+Charge Subcategory
+
+## Description
+
+Indicates what kind of usage or adjustment the row represents.
+
+## Content Constraints
+
+| Constraint      | Value          |
+| :-------------- | :------------- |
+| Column required | True           |
+| Data type       | String         |
+| Allows nulls    | True           |
+| Value format    | list-of-values |
+
+Allowed values for usage:
+
+| Value             | Description                                                                                                                                                                                                                                                                                                                                                |
+| :---------------- | :--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Commitment Used   | Usage charges that are covered by pre-existing commitments, such as capacity commitments or commitment-based discounts.                                                                                                                                                              |
```

Comment:
Why do we want to mix the two here? Wouldn't that not allow people to sum the cost covered by (or not covered by) commitment discounts? or are we saying they'll have to use another column to differentiate this (CommitmentDiscountId?)

Reply:
This may be a carryover from the previous unreviewed CommitmentUtilization draft. 

Agreed the distinction here creates more confusion than clarity. 

1. By distinguishing capacity commitments from commitment-based discounts (which can have a capacity commitment component ie: GCP BigQuery), it implies the capacity commitment does not provide a discount. 
2. AFAIK, usage covered a capacity reservation produces billing data that could be sufficiently captured with on-demand usage charges `chargeCategory == Usage` and `chargeSubcategory == null`
3. The implication is that capacity commitments (with no discounts) could also create charges with `chargeCategory == Usage` and `chargeSubcategory == Commitment Not Used` and I'm not sure that's true

```suggestion
| Commitment Used   | Usage charges associated with the consumption of a commitment-based discount's underlying basis.                                                                                                                              |
```

Reply:
The purpose of the original CommitmentUtilization column was to cover both, so that's why we did it here. Practitioners can use PricingModel == "Commitment-Based Discount" for filtering.

Reply:
Can we use the "Commitment Type" whether "Discount-based" or "Capacity-based". I think we have a dimension for this, the "CommitmentDiscountType", right?

Reply:
Can we use the "Commitment Type" whether "Discount-based" or "Capacity-based". I think we have a dimension for this, the "CommitmentDiscountType", right?

"Charge Type" is "Usage"
"Charge Discount Type" is "Discount-based"
"Charge Subcategory" is "Commitment Used" or "Commitment Unused" or null

"Charge Type" is "Usage"
"Charge Discount Type" is "Capacity-based"
"Charge Subcategory" is "Commitment Used" or "Commitment Unused" or null

Reply:
After further review, my presumption may have been incomplete, I changed the suggestion to cover both discounted and discount-less commitments, but retained vocabulary to use less implied terms such as "covered".  @flanakin @chandra-cloudtrakr 

```suggestion
| Commitment Used   | Usage charges that are associated with consumption of a commitment's underlying basis.                                                                                                                                              |
```